lib/embed/viewer/collection.rb

```diff
@@ -0,0 +1,25 @@
+module Embed
+  class Viewer
+    class Collection < CommonViewer
+
+      def initialize(*args)
+        super
+      end
+
+      def body_html
+        Nokogiri::HTML::Builder.new do |doc|
+          doc.div(class: 'sul-embed-body sul-embed-collection', 'style' => "max-height: #{body_height}px", 'data-sul-embed-theme' => "#{asset_url('geo.css')}") do
```

Comment:
We are embedding the `geo.css` style here, should we just go ahead and stub out the new viewer style?